Extdn/Sniffs/Code/NoDebuggingStatementsSniff.php

```diff
@@ -0,0 +1,62 @@
+<?php
+declare(strict_types=1);
+
+namespace Extdn\Sniffs\Code;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+/**
+ * Class NoDebuggingStatementsSniff
+ * @package Extdn\Sniffs\File
+ */
+class NoDebuggingStatementsSniff implements Sniff
+{
+    /**
+     * @var string
+     */
+    protected $message = 'No debugging statements should be added to production code.';
+
+    /**
+     * @var int
+     */
+    protected $severity = 8;
+
+    /**
+     * @inheritdoc
+     */
+    public function register()
+    {
+        return [T_CLASS];
```

Comment:
Isn't it easier to add here `[T_EXIT, T_STRING, T_PRINT]` and then use PHP CodeSniffer `findNext()` method in `process()`?

Reply:
I'm not sure if it makes it easier :) However, I've never tried it and it seems to be more performant. I refactored it now to this: https://github.com/jissereitsma/extdn-phpcs/commit/603072df810ecb1ed78bffd9500d13c86ace590b#diff-2cc5c6a99e35dbe9dc680a26a915cb02 However, once I register only `[T_EXIT, T_STRING, T_PRINT]` instead of `[T_CLASS]` all warnings are duplicated :(